base/broadcast.jl

```diff
@@ -359,16 +335,9 @@ the following rules:
    (and treats any `Ref` as a 0-dimensional array of its contents and any tuple
    as a 1-dimensional array) expanding singleton dimensions.
```

Comment:
Not from this pull request, but perhaps "If there is at least an array or a `Ref` in the arguments" -> "If there is at least one array or `Ref` in the arguments"? Similarly, perhaps "and treats any `Ref` as a 0-dimensional array of its contents and any tuple as a 1-dimensional array" -> "and treats `Ref`s as 0-dimensional arrays, and tuples as 1-dimensional arrays"?